api/all.yaml

```diff
@@ -582,6 +590,24 @@ paths:
           content:
             application/json:
               schema: {}
+  /temporary-model-credentials:
+    post:
+      operationId: generateTemporaryModelCredentials
+      tags:
+        - TemporaryModelCredentials
+      summary: Generate temporary model credentials.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/GenerateTemporaryModelCredential'
+      responses:
+        '200':
```

Comment:
We don't need to worry about non-200 responses and errors in general?

Reply:
No other endpoints have them defined yet.  That will probably come later.